src/theme/components/socialmedialinks.ts

```diff
@@ -42,73 +42,77 @@ function getColor(color) {
 
 const SocialMediaLinks = defineStyleConfig({
   // The base styles for each part
-  baseStyle: defineStyle({
-    // ul styles should not specifically be identified as ul:
-    display: "flex",
-    alignItems: "flex-start",
-    gap: "s", // 1rem / 16px
-    width: { base: "100%", md: "unset" },
-    marginBottom: "unset",
-    li: {
-      marginInlineEnd: "0",
-      marginTop: "0",
-      width: { base: "100%", md: "unset" },
-      _notFirst: {
-        marginTop: "0",
-      },
-    },
-    a: {
+  baseStyle: defineStyle((props: StyleFunctionProps) => {
+    const { labelsOn, layout } = props;
+    return {
+      // ul styles should not specifically be identified as ul:
       display: "flex",
-      justifyContent: "center",
-      alignContent: "center",
+      alignItems: "flex-start",
+      flexDirection: { base: labelsOn ? "column" : "row", md: layout },
+      justifyContent: { base: "center", md: "unset" },
       flexWrap: "wrap",
-      minWidth: { base: "44px", md: "unset" },
-      minHeight: { base: "44px", md: "unset" },
-      fontStyle: "normal",
-      fontSize: "desktop.body.body2",
-      textDecoration: "none",
-      color: "ui.typography.heading",
-      _hover: {
-        color: "ui.typography.heading",
-        textDecoration: "underline 1px dotted",
+      gap: "s", // 1rem / 16px
```

Comment:
Because of the min-width values that are bring applied, we need to adjust the styles to get the visuals to better match the perceived spacing.  Update the `gap` to the following:

`gap: { base: "0", md: "s" },`